File: Paladins.Api/Paladins.Api/Paladins.Common/Models/ItemsBoughtModel.cs
using Paladins.Common.Interfaces.Definitions;

namespace Paladins.Common.Models
{
    public class ItemsBoughtModel : IModel
    { 
        public int PaladinsItemId { get; set; }
        public string ItemName { get; set; }
        public int ItemLevel { get; set; }
        public int ItemOrder { get; set; }
    }
}

File: Mobile/app/src/main/java/com/fivegearszerochill/paladins/domain/models/match/ItemsBoughtModel.kt
package com.fivegearszerochill.paladins.domain.models.match

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ItemsBoughtModel (
    @Expose
    @SerializedName("paladinsItemId")
    val paladinsItemID: Long,
    @Expose
    @SerializedName("itemName")
    val itemName: String,
    @Expose
    @SerializedName("itemLevel")
    val itemLevel: Long,
    @Expose
    @SerializedName("itemOrder")
    val itemOrder: Long
)